fv3core/pace/fv3core/initialization/geos_wrapper.py

```diff
@@ -114,38 +115,41 @@ def __call__(
         diss_estd: np.ndarray,
     ) -> dict:
```

Comment:
Is there a reason for not making this more strongly typed? Could it be `Dict[str, np.ndarray]`?

Reply:
small oversight, thought I'd done that